discovery/gossiper.go

```diff
@@ -871,6 +871,26 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(nMsg *networkMsg) []l
 			return nil
 		}
 
+		// If this is a local ChannelUpdate announcement, send it to
+		// our peer.
+		if !nMsg.isRemote {
```

Comment:
Shouldn't think be _only_ done in the case that this is the first time we're learning of this channel?